jupyter-book/reproducibility/reproducibility.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>This whole section should come AFTER an introduction to the topic of reproducibility. First we dive into the topic and then we slowly introduce code if required.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>